kde-frameworks/kimageformats/kimageformats-5.84.0-r1.ebuild

```diff
@@ -21,8 +21,16 @@ DEPEND="
 	eps? ( >=dev-qt/qtprintsupport-${QTMIN}:5 )
 	heif? ( >=media-libs/libheif-1.10.0:= )
 	openexr? (
-		media-libs/ilmbase:=
-		media-libs/openexr:=
+		|| (
+			(
+			media-libs/ilmbase:0
+			<media-libs/openexr-3.0.0:0
+			)
+			(
+			dev-libs/imath:3
+			media-libs/openexr:3
+			)
+		)
```

Comment:
@mkszuba: @a17r pointed me to an issue, we've missed with this ebuild. After an upgrade of openexr, no rebuild will be triggered. I'm gonna open a new PR for this.